From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 8 Nov 2017 14:25:43 -0600
Subject: [PATCH] add mock --disable-static option

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 wscript | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/wscript b/wscript
index 334b298..cd5d23c 100644
--- a/wscript
+++ b/wscript
@@ -69,6 +69,11 @@ def options(opt):
 
     opt.samba_add_onoff_option('pthreadpool', with_name="enable", without_name="disable", default=True)
 
+    opt.add_option('--disable-static',
+                   help='unused option, added to avoid %configure (RPM) macro break',
+                   action='store_false',
+                   dest='disable_static')
+
     opt.add_option('--with-system-mitkrb5',
                    help='build Samba with system MIT Kerberos. ' +
                         'You may specify list of paths where Kerberos is installed (e.g. /usr/local /usr/kerberos) to search krb5-config',
