From cbea69b5f5583c73bffea2116537ffea8262f753 Mon Sep 17 00:00:00 2001
From: Gabriel Briones Sayeg <gabriel.briones.sayeg@intel.com>
Date: Thu, 10 Nov 2016 14:00:23 -0600
Subject: [PATCH] confiugre patch to avoid last argument

Signed-off-by: Gabriel Briones Sayeg <gabriel.briones.sayeg@intel.com>
---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index a94c835..afb8acd 100755
--- a/configure
+++ b/configure
@@ -13,5 +13,5 @@ export JOBS
 unset LD_PRELOAD
 
 cd . || exit 1
-${PYTHON:=python} $WAF configure "$@" || exit 1
+${PYTHON:=python} $WAF configure "${@:1:$(($#-1))}" --enable-fhs || exit 1
 cd $PREVPATH
-- 
2.7.4

