From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sat, 23 Mar 2019 16:54:01 +0000
Subject: [PATCH] no man pages

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 buildtools/wafsamba/wafsamba.py | 25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/buildtools/wafsamba/wafsamba.py b/buildtools/wafsamba/wafsamba.py
index 5934f8d..8be4ef1 100644
--- a/buildtools/wafsamba/wafsamba.py
+++ b/buildtools/wafsamba/wafsamba.py
@@ -902,16 +902,6 @@ Build.BuildContext.INSTALL_DIRS = INSTALL_DIRS
 def MANPAGES(bld, manpages, install):
     '''build and install manual pages'''
     bld.env.MAN_XSL = 'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl'
-    for m in manpages.split():
-        source = m + '.xml'
-        bld.SAMBA_GENERATOR(m,
-                            source=source,
-                            target=m,
-                            group='final',
-                            rule='${XSLTPROC} --xinclude -o ${TGT} --nonet ${MAN_XSL} ${SRC}'
-                            )
-        if install:
-            bld.INSTALL_FILES('${MANDIR}/man%s' % m[-1], m, flat=True)
 Build.BuildContext.MANPAGES = MANPAGES
 
 def SAMBAMANPAGES(bld, manpages, extra_source=None):
@@ -921,21 +911,6 @@ def SAMBAMANPAGES(bld, manpages, extra_source=None):
     bld.env.SAMBA_CATALOG = bld.bldnode.abspath() + '/docs-xml/build/catalog.xml'
     bld.env.SAMBA_CATALOGS = 'file:///etc/xml/catalog file:///usr/share/defaults/xml/catalog file://' + bld.env.SAMBA_CATALOG
 
-    for m in manpages.split():
-        source = m + '.xml'
-        if extra_source is not None:
-            source = [source, extra_source]
-        bld.SAMBA_GENERATOR(m,
-                            source=source,
-                            target=m,
-                            group='final',
-                            dep_vars=['SAMBA_MAN_XSL', 'SAMBA_EXPAND_XSL', 'SAMBA_CATALOG'],
-                            rule='''XML_CATALOG_FILES="${SAMBA_CATALOGS}"
-                                    export XML_CATALOG_FILES
-                                    ${XSLTPROC} --xinclude --stringparam noreference 0 -o ${TGT}.xml --nonet ${SAMBA_EXPAND_XSL} ${SRC[0].abspath(env)}
-                                    ${XSLTPROC} --nonet -o ${TGT} ${SAMBA_MAN_XSL} ${TGT}.xml'''
-                            )
-        bld.INSTALL_FILES('${MANDIR}/man%s' % m[-1], m, flat=True)
 Build.BuildContext.SAMBAMANPAGES = SAMBAMANPAGES
 
 @after('apply_link')
