From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sat, 23 Mar 2019 16:54:01 +0000
Subject: [PATCH] no man pages

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 buildtools/wafsamba/wafsamba.py | 29 -----------------------------
 1 file changed, 29 deletions(-)

--- samba-4.17.0/buildtools/wafsamba/wafsamba.py~	2022-08-08 14:15:38.000000000 +0000
+++ samba-4.17.0/buildtools/wafsamba/wafsamba.py	2022-09-15 15:53:30.280457842 +0000
@@ -1164,16 +1164,6 @@
 def MANPAGES(bld, manpages, install):
     '''build and install manual pages'''
     bld.env.MAN_XSL = 'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl'
-    for m in manpages.split():
-        source = m + '.xml'
-        bld.SAMBA_GENERATOR(m,
-                            source=source,
-                            target=m,
-                            group='final',
-                            rule='${XSLTPROC} --xinclude -o ${TGT} --nonet ${MAN_XSL} ${SRC}'
-                            )
-        if install:
-            bld.INSTALL_FILES('${MANDIR}/man%s' % m[-1], m, flat=True)
 Build.BuildContext.MANPAGES = MANPAGES
 
 def SAMBAMANPAGES(bld, manpages, extra_source=None):
@@ -1183,25 +1173,6 @@
     bld.env.SAMBA_CATALOG = bld.bldnode.abspath() + '/docs-xml/build/catalog.xml'
     bld.env.SAMBA_CATALOGS = 'file:///etc/xml/catalog file:///usr/local/share/xml/catalog file://' + bld.env.SAMBA_CATALOG
 
-    for m in manpages.split():
-        source = [m + '.xml']
-        if extra_source is not None:
-            source = [source, extra_source]
-        # ${SRC[1]}, ${SRC[2]} and ${SRC[3]} are not referenced in the
-        # SAMBA_GENERATOR but trigger the dependency calculation so
-        # ensures that manpages are rebuilt when these change.
-        source += ['build/DTD/samba.build.pathconfig', 'build/DTD/samba.entities', 'build/DTD/samba.build.version']
-        bld.SAMBA_GENERATOR(m,
-                            source=source,
-                            target=m,
-                            group='final',
-                            dep_vars=['SAMBA_MAN_XSL', 'SAMBA_EXPAND_XSL', 'SAMBA_CATALOG'],
-                            rule='''XML_CATALOG_FILES="${SAMBA_CATALOGS}"
-                                    export XML_CATALOG_FILES
-                                    ${XSLTPROC} --xinclude --stringparam noreference 0 -o ${TGT}.xml --nonet ${SAMBA_EXPAND_XSL} ${SRC[0].abspath(env)}
-                                    ${XSLTPROC} --nonet -o ${TGT} ${SAMBA_MAN_XSL} ${TGT}.xml'''
-                            )
-        bld.INSTALL_FILES('${MANDIR}/man%s' % m[-1], m, flat=True)
 Build.BuildContext.SAMBAMANPAGES = SAMBAMANPAGES
 
 @after('apply_link')
