--- samba-samba-4.10.0/buildtools/wafsamba/wafsamba.py~	2019-03-23 14:19:23.000000000 +0000
+++ samba-samba-4.10.0/buildtools/wafsamba/wafsamba.py	2019-03-23 16:37:02.665549543 +0000
@@ -904,16 +904,6 @@
 def MANPAGES(bld, manpages, install):
     '''build and install manual pages'''
     bld.env.MAN_XSL = 'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl'
-    for m in manpages.split():
-        source = m + '.xml'
-        bld.SAMBA_GENERATOR(m,
-                            source=source,
-                            target=m,
-                            group='final',
-                            rule='${XSLTPROC} --xinclude -o ${TGT} --nonet ${MAN_XSL} ${SRC}'
-                            )
-        if install:
-            bld.INSTALL_FILES('${MANDIR}/man%s' % m[-1], m, flat=True)
 Build.BuildContext.MANPAGES = MANPAGES
 
 def SAMBAMANPAGES(bld, manpages, extra_source=None):
@@ -923,21 +913,6 @@
     bld.env.SAMBA_CATALOG = bld.srcnode.abspath() + '/bin/default/docs-xml/build/catalog.xml'
     bld.env.SAMBA_CATALOGS = 'file:///etc/xml/catalog file:///usr/share/defaults/xml/catalog file://' + bld.env.SAMBA_CATALOG
 
-    for m in manpages.split():
-        source = m + '.xml'
-        if extra_source is not None:
-            source = [source, extra_source]
-        bld.SAMBA_GENERATOR(m,
-                            source=source,
-                            target=m,
-                            group='final',
-                            dep_vars=['SAMBA_MAN_XSL', 'SAMBA_EXPAND_XSL', 'SAMBA_CATALOG'],
-                            rule='''XML_CATALOG_FILES="${SAMBA_CATALOGS}"
-                                    export XML_CATALOG_FILES
-                                    ${XSLTPROC} --xinclude --stringparam noreference 0 -o ${TGT}.xml --nonet ${SAMBA_EXPAND_XSL} ${SRC[0].abspath(env)}
-                                    ${XSLTPROC} --nonet -o ${TGT} ${SAMBA_MAN_XSL} ${TGT}.xml'''
-                            )
-        bld.INSTALL_FILES('${MANDIR}/man%s' % m[-1], m, flat=True)
 Build.BuildContext.SAMBAMANPAGES = SAMBAMANPAGES
 
 @after('apply_link')
