--- samba-samba-4.5.1/source4/heimdal/lib/wind/generate.py~	2016-10-24 19:44:07.000000000 +0000
+++ samba-samba-4.5.1/source4/heimdal/lib/wind/generate.py	2017-01-15 18:43:38.853480305 +0000
@@ -45,9 +45,7 @@
         self.name  = os.path.basename(name)
         self.file  = open(name, 'w')
         self.file.write('/* ' + name + ' */\n')
-        self.file.write('/* Automatically generated at ' +
-                        datetime.datetime.now().isoformat() +
-                        ' */\n\n')
+        self.file.write('/* Automatically generated */\n\n')
     
     def close(self) :
         """End and close the file header"""
